backend/nest-app/src/auth/auth.service.ts

```diff
@@ -1,12 +1,19 @@
+import { MikroORM } from '@mikro-orm/core';
 import { Injectable } from '@nestjs/common';
+import { User } from 'src/users/users.entity';
+import { UsersService } from 'src/users/users.service';
 
 @Injectable()
 export class AuthService {
-  signup() {
-    return { msg: 'Signing Up' };
-  }
+  constructor(private usersService: UsersService) {}
 
-  signin() {
-    return { msg: 'Signing In' };
+  async validateUser(email: string, pass: string): Promise<any> {
```

Comment:
it highly encouraged not to use `any` types. Like for you case:
```suggestion
  async validateUser(email: string, pass: string): Promise<Omit<User, 'password'> | null> {
```